### DISCUSSION TITLE
How to handle CARP IP adresses + normal IPs on same interface?

### DISCUSSION THREAD

**Discussion by EHRETic** [netbox-community/netbox#16285] [Q&A] [answered]
Hello everyone,

I need help to expand my skills with Netbox.
I recently installed an OPNsense HA cluster and am using CARP IPs.

As a result, for a named interface (both FWs), I have 3 IPs:
- X.X.X.1 --> CARP IP (so shared)
- X.X.X.2 --> FW 1 only
- X.X.X.3 --> FW 2 only

How do I configure this in Netbox?
I know I should use FHRP groups but before putting a mess in my system, I'd like to have a little feedback from specialists...

Thanks in advance for you help! ;-)

**Comment by candlerb** [comment, marked as answer]
It's pretty straightforward.
* Add IP address X.X.X.2 to the interface on the first device
* Add IP address X.X.X.3 to the interface on the second device
* Create an FHRP group with address X.X.X.1
* Assign it to both interfaces

The GUI isn't entirely obvious for how to do this. It used to be that from the device's interface list, instead of clicking "Edit" on an interface, you have to "View" the interface, then you get the list of FHRP groups which you can assign. But there's another way now where you can click '+' and then "Assign FHRP group"

**Reply by EHRETic** [reply]
@candlerb thanks a lot, I was half way, but I missed a few things to make it right! :-)

Enjoy your weekend
